pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -33,7 +33,7 @@ type ApplicationBackupSpec struct {
 	PlatformCredential string                             `json:"platformCredential"`
 	RancherProjects    map[string]string                  `json:"rancherProjects"`
 	Selectors          map[string]string                  `json:"selectors"`
-	NamespaceSelector  map[string]string                  `json:"namespaceSelector"`
+	NamespaceSelector  string                             `json:"namespaceSelector"`
```

Comment:
Remove trailing space here

Reply:
This trailing space is due to the go fmt